ui/home/game.players.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { CivDef, GAMES, Game, GamePlayer, SteamProfileMap } from 'pydt-shared';
+
+
+@Component({
+    selector: 'pydt-game-players',
+    templateUrl: './game.players.html',
+    styleUrls: ['./game.players.css']
+})
+
+export class GamePlayers implements OnInit {
```

Comment:
The files for this should be gamePlayers.component.ts/html/css, and the class name should be GamePlayersComponent to match the convention, also make sure you run `npm run lint` and address any linting issues that come up, thanks!